rllib/tuned_examples/dqn/stateless-cartpole-r2d2.yaml

```diff
@@ -2,17 +2,18 @@ stateless-cartpole-r2d2:
     env: ray.rllib.examples.env.stateless_cartpole.StatelessCartPole
     run: R2D2
     stop:
-        episode_reward_mean: 150
-        timesteps_total: 1000000
+        episode_reward_mean: 100
```

Comment:
Actually, increasing `num_workers` to e.g. 3 or 4 also may work here to speed up learning.

Again, just making 100% sure we didn't introduce a bigger learning regression.